src/Controller/Component/PrgComponent.php

```diff
@@ -11,14 +11,16 @@ class PrgComponent extends Component
      * Default config
      *
      * ### Options
-     * - `actions` Method name(s) of actions to use PRG. Or bool for all or none.
-     *   You can pass a single action as string or multiple as array. Default is
-     *   true and all actions will be processed by the component.
+     * - `actions` : Action name(s) to use PRG for. You can pass a single action
+     *   as string or multiple as array. If boolean `true` all actions will be
+     *   processsed if `false` none. Default is ['index'].
+     * - `queryStringToData` : Set query string as request data. Default `true`.
      *
      * @var array
      */
     protected $_defaultConfig = [
-        'actions' => true,
+        'actions' => ['index'],
```

Comment:
Yes `actions` is configurable`. I personally like to avoid reading too much stuff from `Configure`, you end up almost like using globals. Since `lookup` is a crud actions it's a sane default to have so that all FOC plugins inter-operate easily.